launch/topic_switch.py

```diff
@@ -11,18 +11,30 @@ def generate_launch_description():
     message_type_argument = DeclareLaunchArgument(
         "message_type",
         default_value="std_msgs.msg.String",
-        description="topic message type, like 'std_msgs.msg.String'",
+        description="topic message type",
+    )
+    channels_argument = DeclareLaunchArgument(
+        "channels",
+        default_value="2",
+        description="number of desired channels",
     )
 
     topic_switch_node = Node(
         package="ezrassor_topic_switch",
         node_executable="topic_switch",
```

Comment:
ros2 foxy is giving me a warning:
> The parameter 'node-executable' is deprecated, use 'executable' instead

Super easy fix thankfully

Reply:
Will fix this when I bump all the packages to Foxy, currently building for Dashing and the new param names arent available.